src/blocks/card.scss

```diff
@@ -0,0 +1,157 @@
+main {
+  display: grid;
+
+  padding: 50px 40px;
+  min-width: 320px;
+  max-width: 1200px;
+  margin: 0 auto;
+
+  @include grid-columns(4);
+
+  grid-template-rows: repeat(2, 408px);
+  grid-gap: 46px 48px;
+
+  justify-content: center;
+  background-color: $main-background-color;
+}
+
+@media (min-width: 320px) {
+  .product-cards {
+    @include grid-columns(1);
+  }
+}
+
+@media (min-width: 488px) {
+  .product-cards {
+    @include grid-columns(2);
+  }
+}
+
+@media (min-width: 768px) {
+  .product-cards {
+    @include grid-columns(3);
+  }
+}
+
+@media (min-width: 1024px) {
+  .product-cards {
+    @include grid-columns(4);
+  }
+}
+
+.product-card {
+  box-sizing: border-box;
+  padding-inline: 16px;
+  background-color: $main-background-color;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  transform: scale(1);
+  transition: transform 300ms;
+
+  &__image {
+    object-position: center;
+    margin-inline: 3px;
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__product-title {
+    margin-top: 40px;
+    margin-bottom: 0;
+    height: 36px;
+    font-size: 12px;
+    line-height: 18px;
+    text-align: left;
+    font-weight: 500;
+    color: $bold-font-color;
+  }
+
+  &__product-code {
+    align-self: start;
+    margin-top: 4px;
+    margin-bottom: 0;
+    width: 98px;
+    font-weight: 400;
+    font-size: 10px;
+    height: 14px;
+    line-height: 14px;
+    color: $main-font-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+    height: 16px;
+    margin-top: 16px;
+  }
+
+  &__price {
+    height: 18px;
+    margin-top: 24px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    &-name {
+      color: $main-font-color;
+      margin: 0;
+      width: 31px;
+      font-size: 12px;
+      line-height: 18px;
+      font-weight: 400px;
```

Comment:
The `font-weight` property should not have a unit. It should be `font-weight: 400;` instead of `font-weight: 400px;`.